pallets/route-executor/src/lib.rs

```diff
@@ -724,29 +720,14 @@ impl<T: Config> Pallet<T> {
 	}
 
 	fn validate_sell(route: Vec<Trade<T::AssetId>>, amount_in: T::Balance) -> Result<T::Balance, DispatchError> {
+		// Validate that the route is properly structured
 		let asset_in = route.first().ok_or(Error::<T>::InvalidRoute)?.asset_in;
 		let asset_out = route.last().ok_or(Error::<T>::InvalidRoute)?.asset_out;
 
-		with_transaction::<T::Balance, DispatchError, _>(|| {
```

Comment:
What if the route is invalid, because of low liquidity etc? We wanna allow to set route for such?

Reply:
low liquidity situations are replicated by calculation